st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -97,7 +97,8 @@ def get_one(self, name, requester_user, scope=None, user=None, decrypt=False):
         key_ref = get_key_reference(scope=scope, name=name, user=user)
         extra = {"scope": scope, "name": name, "user": user, "key_ref": key_ref}
         LOG.debug("GET /v1/keys/%s", name, extra=extra)
-
+        LOG.audit("User %s decrypted the value %s ", user, name,
+                  extra={"user": user, "scope": scope, "key_name": name, "operation": "decrypt"})
```

Comment:
Could you also reformat the code/syntax with black?

That's the only blocker that's failing the CI check:
https://github.com/StackStorm/st2/runs/5770402290?check_suite_focus=true#step:11:101